packages/block-library/src/post-comment-content/index.php

```diff
@@ -18,10 +18,14 @@ function render_block_core_post_comment_content( $attributes, $content, $block )
 		return '';
 	}
 	$wrapper_attributes = get_block_wrapper_attributes();
+	$comment_text       = get_comment_text( $block->context['commentId'] );
+	if ( ! $comment_text ) {
+		return;
```

Comment:
```suggestion
		return '';
```

Nit: To keep return type consistent.